languages/ruby/concepts/nil/about.md

```diff
@@ -1 +1,63 @@
-TODO: add information on nil concept
+## Key Points:
+
+- When a class' `.new` method is called to create an object instance, the `.initialize` method is passed all arguments to initialize the instance's state.
+- instance variable names are prefixed with `@`.
+- instance variables default to `nil` until they are explicitly set.
+- instance variables are private by default, and they should be manipulated with getters and setters
```

Comment:
There is no such thing as a "public instance variable" only that they can be publicly exposed.

Reply:
My thought here is that perhaps we remove "by default".